src/deluge/modulation/arpeggiator.h

```diff
@@ -120,6 +142,7 @@ class Arpeggiator final : public ArpeggiatorBase {
 	bool hasAnyInputNotesActive();
 
 	OrderedResizeableArray notes;
+	ResizeableArray notesAsPlayed;
```

Comment:
If I understand correctly, `notes` tracks the currently sounding notes generated by the arpegiator while `notesAsPlayed` tracks the "live" notes from the underlying sequence? It would be good to describe this in detail in `///` doc comments here.